defaults/main/drop_lasso.yml

```
@@ -0,0 +1,25 @@
+---
+
+
+pf_drop_lasso_template: default-drop-lasso.j2
+
+# this is IP4 and IPv6 respectively
+# see https://www.spamhaus.org/drop/
+#
+pf_drop_lasso_v4: true
+pf_drop_lasso_v6: true
+
```

The configuration can be simplified further

```suggestion
pf_drop_lasso_rcconf:
  - { name: daily_pf_droplasso_enable, value: "{{ pf_drop_lasso_enable }}" }
  - { name: daily_pf_droplasso_v4, value: "{{ pf_drop_lasso_v4 | bool | ternary('YES', 'NO') }}" }
  - { name: daily_pf_droplasso_v6, value: "{{ pf_drop_lasso_v6 | bool | ternary('YES', 'NO') }}" }
```